### PR TITLE
Use forta ipfs node for info command

### PR DESCRIPTION
**Pull request by Robert-H-Leonard**
### Info command when botId has no ipfs data:

Screen Shot 2022-06-27 at 11 26 14 AM

### Info command when botId has ipfs data:

Screen Shot 2022-06-27 at 11 27 46 AM

### CODE REVIEW

File: cli/utils/ipfs/get.from.ipfs.ts [hunk 1]
```diff
@@ -21,17 +21,28 @@ export interface IpfsManifestData {
...
+    ipfsHttpClient: AxiosInstance,
+    agentId: string
 ): GetFromIpfs {
     assertExists(ipfsHttpClient, 'ipfsHttpClient')
+    assertIsNonEmptyString(agentId, 'agentId')
 
     return async function getIpfsByHash(metadataHash: string) {
-        const { data } = await ipfsHttpClient.get(`/ipfs/${metadataHash}`);
+        try {
+            const { data } = await ipfsHttpClient({url: `/ipfs/${metadataHash}`, baseURL: "https://ipfs.forta.network"});
```
Comment (haseebrabbani) [inline cli/utils/ipfs/get.from.ipfs.ts:32]: lets use dependency injection to inject a `fortaIpfsHttpClient` (which is configured to point to ipfs.forta.network)

File: cli/utils/ipfs/get.from.ipfs.ts [hunk 2]
```diff
@@ -21,17 +21,28 @@ export interface IpfsManifestData {
...
-        }
+            if(data && data.manifest) {
+                return data;
+            }
+
+            throw Error(`No data found for ipfs hash ${metadataHash}`)
 
-        throw Error(`No data found for ipfs hash ${metadataHash}`)
+        } catch(e) {
+            if((e.message as string).includes('status code 403')) {
```
Comment (haseebrabbani) [inline cli/utils/ipfs/get.from.ipfs.ts:41]: ideally we check the error code itself i.e. `e.response.status`

File: cli/utils/ipfs/get.from.ipfs.ts (hunk 1 above)
Comment (Robert-H-Leonard) [reply cli/utils/ipfs/get.from.ipfs.ts]: Updated.

File: cli/utils/ipfs/get.from.ipfs.ts (hunk 2 above)
Comment (Robert-H-Leonard) [reply cli/utils/ipfs/get.from.ipfs.ts]: Updated this.

File: cli/di.container.ts [hunk 3]
```diff
@@ -259,6 +259,10 @@ export default function configureContainer(args: any = {}) {
         }
       }
       return axios.create(options)
+    }).singleton(),
+    fortaIpfsHttpClient: asFunction((ipfsGatewayUrl: string, ipfsGatewayAuth: string) => {
```
Comment (haseebrabbani) [inline cli/di.container.ts:263]: we can remove these 2 parameters since they are not used inside the function

**Review by haseebrabbani — APPROVED** [review]

**Pull request merged**